source/statements.tex

```diff
@@ -146,6 +146,19 @@
 \indextext{label!\idxcode{default}}%
 Case labels and default labels shall occur only in \keyword{switch} statements.
 
+\pnum
+\indextext{statement!control-flow-limited}%
+A \defn{control-flow-limited statement} is a statement \tcode{S} for which:
```

Comment:
```suggestion
A \defnadj{control-flow-limited}{statement} is a statement \tcode{S} for which:
```

Reply:
Agreed. With this change, `\indextext{statement!control-flow-limited}%` should go, because it's now automatic / implied.